evalai/add_token.py

```diff
@@ -16,25 +17,26 @@ def set_token(auth_token):
     Configure EvalAI Token.
     """
     """
-    Invoked by `evalai settoken <your_evalai_auth_token>`.
+    Invoked by `evalai set_token <your_evalai_auth_token>`.
     """
     if validators.length(auth_token, min=LEN_OF_TOKEN, max=LEN_OF_TOKEN):
         if not os.path.exists(AUTH_TOKEN_DIR):
             os.makedirs(AUTH_TOKEN_DIR)
-        with open(AUTH_TOKEN_PATH, "w+") as fw:
-            try:
-                auth_token = {"token": "{}".format(auth_token)}  # noqa
-                auth_token = json.dumps(auth_token)
-                fw.write(auth_token)
-            except (OSError, IOError) as e:
-                echo(e)
-            echo(
-                style(
-                    "Success: Authentication token is successfully set.",
-                    bold=True,
-                    fg="green",
+        if is_auth_token_valid(auth_token):
```

Comment:
Please add a else block showing the error message from API that `Token is not valid`